node/src/actors/chain_manager/mod.rs

```diff
@@ -1394,11 +1397,17 @@ impl ChainManager {
                         }
 
                         if let Some(consolidated_superblock) = act.chain_state.superblock_state.get_current_superblock() {
+                            let sb_hash = consolidated_superblock.hash();
                             // Let JSON-RPC clients know that the blocks in the previous superblock can now
                             // be considered consolidated
                             act.notify_superblock_consolidation(consolidated_superblock);
 
-                            log::info!("Consensus reached for Superblock #{}", voted_superblock_beacon.checkpoint);
+                            log::info!("Consensus reached for Superblock #{} with {} out of {} votes. Committee size: {}",
+                                       voted_superblock_beacon.checkpoint,
+                                       act.chain_state.superblock_state.votes_counter_from_superblock(&sb_hash),
+                                       act.chain_state.superblock_state.get_current_superblock_votes().len(),
```

Comment:
This call creates a temporary `HashSet` which is not needed because we only need to count the votes. There is a similar method called `valid_votes_counter` which should return the exact same value.

```suggestion
                                       act.chain_state.superblock_state.valid_votes_counter(),
```